api/app/postgresql.js

```diff
@@ -0,0 +1,58 @@
+const items = require("../db/items.json");
+const { Pool } = require("pg");
+
+const pool = new Pool({
+  user: process.env.POSTGRES_USER || "kumademo",
+  host: process.env.POSTGRES_HOST || "localhost",
+  database: process.env.POSTGRES_DB || "kumademo",
+  password: process.env.POSTGRES_PASSWORD || "kumademo",
+  port: process.env.POSTGRES_PORT_NUM || 5432,
```

Comment:
Can you drop a comment here that we cannot use POSTGRES_PORT, because it is taken on K8S. So we won't make the same mistake

Reply:
added comment beside it